src/features/video/videoSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+
+const initialState ={
+    videos:[],
+    videoLoader:false,
+}
+
+export const getVideoList = createAsyncThunk('videoList/getVideoList',async (_,{rejectWithValue})=>{
+    try {
+        const res = await axios.get("/api/videos");
+        console.log(res.data.videos);
+        return res.data.videos;
+    } catch (error) {
+        console.log(error);
+        return rejectWithValue(error.res.data);
```

Comment:
Did not knew about the error format here too!!Will check the format of it as an object

Reply:
Its mentioned in the doc( [https://redux-toolkit.js.org/api/createAsyncThunk](url) ) as think API which is an object containing this parameter. like thunkApi.rejectWithValue